extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.kafka.streams.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "kafka-streams", phase = ConfigPhase.RUN_TIME)
+public class KafkaStreamsRuntimeConfig {
+
+    /**
+     * A unique identifier for this Kafka Streams application.
+     */
+    @ConfigItem
+    public String applicationId;
+
+    /**
+     * A comma-separated list of host:port pairs identifying the Kafka bootstrap server(s)
+     */
+    @ConfigItem(defaultValue = "localhost:9012")
+    public String bootstrapServers;
```

Comment:
Would a `List<InetSocketAddress>` be more appropriate here?  That would give some base level of validation.  Or at a minimum a `List<String>`?

Reply:
Would a value like "host1:9012,host2:9012" automatically parsed into `List<InetSocketAddress>`? Then it seems like a good thing to do; `List<String>` doesn't seem overly useful, would not essentially every value qualify as such? Note the value must be passed as a string to Kafka Streams, so the conversion will be a bit of a forth and back, but if `List<InetSocketAddress>` works, why not.

Reply:
> Would a value like "host1:9012,host2:9012" automatically parsed into `List<InetSocketAddress>`?

Yes, exactly.

> Note the value must be passed as a string to Kafka Streams, so the conversion will be a bit of a forth and back, but if `List<InetSocketAddress>` works, why not.

It'll be some free fail-fast validation.  At some point we'll have a consolidated config validation blob that will print out nice errors like "xxxx is not a valid value for properties of type `InetSocketAddress`" or something like that.  You'll be ready for it!

Reply:
Done.